### PR TITLE
Add Adventurous Machines as sponsor

### PR DESCRIPTION
**Pull request by antoinepetty**

### CODE REVIEW

**Comment by antoinepetty** [comment]
(sorry for the typo in the branch name)

**Review by BK-211 — APPROVED** [review]

File: _config.yml [hunk 1]
```diff
@@ -72,6 +72,10 @@ sponsors:
     link: //www.southampton.ac.uk/
     image: /images/sponsors/university-of-southampton.svg
 
+  - name: Adventurous Machines
```
Comment (RealOrangeOne) [inline _config.yml:75]: Question: What order should sponsors be in? Alphabetical? Order of tier? Aesthetic appeal of the logo?

Comment (BK-211) [reply _config.yml:75]: This issue came up at the last fundraising meeting and wasnt't fully resolved. Until it is I'd say put the logos in the date order that they have offered sponsorship?

File: images/sponsors/adventurous-machines.svg [hunk 2]
```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (raccube) [inline images/sponsors/adventurous-machines.svg:1]: This can be minified.

**Review by raccube — CHANGES_REQUESTED** [review]

File: images/sponsors/adventurous-machines.svg (hunk 2 above)
Comment (raccube) [reply images/sponsors/adventurous-machines.svg:1]: This applies to the whole file. There's a lot of Inkscape specific stuff in here we don't need for web.

File: _config.yml (hunk 1 above)
Comment (RealOrangeOne) [reply _config.yml:75]: Ah ok. Yup that makes sense to me! :+1:

**Review by raccube — APPROVED** [review]

**Review by sedders123 — APPROVED** [review]

**Pull request merged**